azure-mgmt-resources/src/main/java/com/microsoft/azure/management/resources/fluentcore/arm/collection/SupportsBatchDeletion.java

```diff
@@ -8,40 +8,46 @@
 
 import java.util.Collection;
 
+import com.microsoft.azure.management.apigeneration.Beta;
 import com.microsoft.azure.management.apigeneration.LangDefinition;
 import com.microsoft.azure.management.apigeneration.LangDefinition.MethodConversion;
 import rx.Observable;
 
 /**
  * Provides access to deleting multiple resource from Azure, identifying them by their IDs.
- *
- * (Note: this interface is not intended to be implemented by user code)
+ * <p>
+ * (Note this interface is not intended to be implemented by user code.)
  */
 @LangDefinition(ContainerName = "CollectionActions", CreateAsyncMethods = true, MethodConversionType = MethodConversion.OnlyMethod)
+@Beta
 public interface SupportsBatchDeletion {
     /**
      * Deletes the specified resources from Azure asynchronously and in parallel.
      * @param ids resource IDs of the resources to be deleted
      * @return an observable from which all of the successfully deleted resources can be observed
      */
+    @Beta
     Observable<String> deleteByIdsAsync(Collection<String> ids);
```

Comment:
Why `deleteByIdsAsync` is beta? it's [implementation](https://github.com/Azure/azure-sdk-for-java/blob/ebcb486b6f4cb581918a38cb02a3bf48df263662/azure-mgmt-resources/src/main/java/com/microsoft/azure/management/resources/fluentcore/arm/collection/implementation/TopLevelModifiableResourcesImpl.java#L79) uses `Observable`s not `Completable`. Am i missing anything?